app/gulpfile.js

```diff
@@ -19,29 +23,36 @@ gulp.task('buildMap', function(){
     browserifyShare('Map');
 });
 
-gulp.task('watch', function() {
+gulp.task('watch-app', function() {
```

Comment:
Maybe define a watcher for each specific part of the code:
- `serverWatcher` in `watch-app` should only look at `./server/app.js` and the same for `admin`
- maybe an independant dbWatcher, so that we can define a global watcher in the case when we need to serve and watch both app and admin servers